urbandict-search/README.md

```diff
@@ -296,12 +294,13 @@ fl.search(read_query_data(text), callback=ppr, topk=top_k)
 def print_topk(resp, word):
     for d in resp.search.docs:
         print(f'Ta-Dah🔮, here are what we found for: {word}')
-        for idx, kk in enumerate(d.topk_results):
-            score = kk.score.value
+        for idx, match in enumerate(d.matches):
+            score = match.score.value
             if score <= 0.0:
                 continue
-            print('{:>2d}:({:f}):{}'.format(
-                idx, score, kk.match_doc.buffer.decode()))       
+            doc = match.chunks[0].text
+            word, word_def = doc.split('+-=', maxsplit=1)
```

Comment:
Need to adapt it to the actuall app.py code

Reply:
@YueLiu-jina Would you please fix this?